### PR TITLE
fix: normalizer/packageStatusLegend issue

### PR DESCRIPTION
**Pull request by shleewhite**
Fixes several bugs:
* figmaStatus, designCommitteeReview, engineerCommitteeReview props can be null - fixes when pages that these don't apply to getting the badges
* badges don't stack next to the header on narrow screen - instead they go below the header as designed
* renames badgePopover folder to badge-popover - fixes webpack error
* pass the correct props in the componentHeader stories

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [ ] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Review by andipants12 — APPROVED** [review]
👍 For the bug fix.

**Review by nkrantz — APPROVED** [review]

File: packages/paste-website/src/utils/GraphqlUtils.ts [hunk 1]
```diff
@@ -63,12 +63,8 @@ const getMutation = (key: string): MutationFunction => {
     case 'allAirtable': {
       return ({status, Figma, ...rest}) => ({
         packageStatus: sentenceCase(status),
-        figmaStatus: Figma === null ? undefined : Figma,
```
Comment (zahnster) [inline packages/paste-website/src/utils/GraphqlUtils.ts:66]: We don't need the null check now?

Comment (shleewhite) [reply packages/paste-website/src/utils/GraphqlUtils.ts:66]: No, the packageStatusLegend checks if it's null or not now. This is what caused the problem.

**Review by zahnster — APPROVED** [review]
👍🏼 !

**Pull request merged**